go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/charmbracelet/lipgloss v0.9.1
 	github.com/client9/gospell v0.0.0-20160306015952-90dfc71015df
 	github.com/confluentinc/ccloud-sdk-go-v1-public v0.0.0-20230427001341-5f8d2cce5ad9
+	github.com/confluentinc/ccloud-sdk-go-v2-internal/srcmv3 v0.0.1
```

Comment:
Make sure you use confluentinc/ccloud-sdk-go-v2 (public)!

Reply:
srcm/v3 is not in the public repo yet... looks like this PR will have to wait 😅